build.gradle

```diff
@@ -86,110 +87,80 @@ test {
 }
 
 publishing {
-  afterEvaluate {
-    def pluginUrl = 'https://github.com/sonatype-nexus-community/scan-gradle-plugin'
-
-    pluginBundle {
-      website = pluginUrl
-      vcsUrl = pluginUrl
-      description = pluginDescription
-      tags = ['sonatype', 'scan', 'dependencies', 'ossindex', 'iq server']
-      System.setProperty('gradle.publish.key', System.getenv('GRADLE_PUBLISH_KEY') ?: '')
-      System.setProperty('gradle.publish.secret', System.getenv('GRADLE_PUBLISH_SECRET') ?: '')
-    }
```

Comment:
We need to keep this part so the plugin is also listed in the Gradle plugins portal:
https://plugins.gradle.org/plugin/org.sonatype.gradle.plugins.scan